.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should this file be here?

Reply:
@0legIv you can add this to the gitignore